docs/reference/slm/apis/slm-delete.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[slm-api-delete-policy]]
+=== Delete snapshot lifecycle policy API
+++++
+<titleabbrev>Delete policy</titleabbrev>
+++++
+
+Deletes an existing snapshot lifecycle policy.
+
+[[slm-api-delete-lifecycle-request]]
+==== {api-request-title}
+
+`DELETE /_slm/policy/<snapshot-lifecycle-policy-id>`
+
+[[slm-api-delete-lifecycle-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage_slm`
```

Comment:
I don't think a list is needed if there's only one item. Don't feel strongly though.
```suggestion
If the {es} {security-features} are enabled, you must have the `manage_slm`
```